awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/OtelJMXMetricsCustomizerProvider.java

```diff
@@ -0,0 +1,144 @@
+
+            logger.info("Otel JMX metric collection enabled");
+        }
+        return sdkMeterProviderBuilder;
+    }
+
+    private static void configureMetricFilter(
+            ConfigProperties configProps,
+            SdkMeterProviderBuilder sdkMeterProviderBuilder,
+            Set<String> registeredScopeNames) {
+        Set<String> exporterNames =
+                DefaultConfigProperties.getSet(configProps, "otel.metrics.exporter");
+        if (exporterNames.contains("none")) {
+            for (String scope : registeredScopeNames) {
+                sdkMeterProviderBuilder.registerView(
+                        InstrumentSelector.builder().setMeterName(scope).build(),
+                        View.builder().setAggregation(Aggregation.defaultAggregation()).build());
+
+                logger.log(Level.FINE, "Registered scope {0}", scope);
+            }
+            sdkMeterProviderBuilder.registerView(
+                    InstrumentSelector.builder().setName("*").build(),
+                    View.builder().setAggregation(Aggregation.drop()).build());
+        }
+    }
+
+    private static Duration getMetricExportInterval(ConfigProperties configProps) {
+        Duration exportInterval =
+                configProps.getDuration("otel.metric.export.interval", DEFAULT_METRIC_EXPORT_INTERVAL);
+        logger.log(Level.FINE, String.format("Otel JMX Metrics export interval: %s", exportInterval));
+        // Cap export interval to 60 seconds. This is currently required for metrics-trace correlation
+        // to work correctly.
+        if (exportInterval.compareTo(DEFAULT_METRIC_EXPORT_INTERVAL) > 0) {
+            exportInterval = DEFAULT_METRIC_EXPORT_INTERVAL;
+            logger.log(
+                    Level.INFO,
+                    String.format("Otel JMX Metrics export interval capped to %s", exportInterval));
+        }
+        return exportInterval;
+    }
+
+    private enum JMXExporterProvider {
+        INSTANCE;
+
+        public MetricExporter createExporter(ConfigProperties configProps) {
+            String protocol =
+                    OtlpConfigUtil.getOtlpProtocol(OtlpConfigUtil.DATA_TYPE_METRICS, configProps);
+            logger.log(Level.FINE, String.format("Otel JMX metrics export protocol: %s", protocol));
+
+            String otelJMXEndpoint;
+            if (protocol.equals(OtlpConfigUtil.PROTOCOL_HTTP_PROTOBUF)) {
+                otelJMXEndpoint = configProps.getString(OTEL_JMX_ENDPOINT_CONFIG, "http://localhost:4314");
+                logger.log(
+                        Level.FINE, String.format("Otel JMX metrics export endpoint: %s", otelJMXEndpoint));
+                return OtlpHttpMetricExporter.builder()
+                        .setEndpoint(otelJMXEndpoint)
+                        .setDefaultAggregationSelector(this::getAggregation)
+                        .setAggregationTemporalitySelector(CloudWatchTemporalitySelector.alwaysDelta())
```

Comment:
We want to use the default temporality selector which prefers cumulative metrics to delta since AMP will be supported.

Reply:
removed the override, by default the sdk sets it to cumulative - https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/http/metrics/OtlpHttpMetricExporterBuilder.java#L44C24-L44C54